src/style.css

```diff
@@ -1 +1,25 @@
 /* styles go here */
+
+.infoAbout1Form {
+  margin-bottom: 10px;
+}
+
+.infoAboutRegistr {
+  padding-top: 10px;
+}
+
+.interestFact {
+  margin-bottom: 10px;
+}
+
+.selector {
+  overflow: hidden;
+}
+
+.infoAboutForm {
+  margin-bottom: 20px;
+}
```

Comment:
I wrote about it previous time, don't create class for every container, just create one and reuse it

Reply:
if talk about: 'I still don't think you need this style', without it, test doesn't pass